### DISCUSSION TITLE
About the Pause Option Menu....

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#94] [Source Q&A]
just realized that beyond the point after week 3, it will begin crashing the game for some reason, but in weeks 2 & 3, the background just disappears when going back, while week 1 is perfectly fine, anyone know how to fix that?

**Comment by Theoyeah** [comment]
ima check that

**Reply by ThatOneSillyGuyDon** [reply]
> ima check that

possible file I suspect: altoptions/PauseOptionsState or something with PlayState, but that's all i suspect

**Reply by Theoyeah** [reply]
oh fuck, didn't remember that i reset my pc 5 days ago, so compiling is lllooooonnngnggg
